index.html

```diff
@@ -8,310 +8,339 @@
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="buttons">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <p class="invite">See Project</p>
+        </div>
+        <div class="art-container six">
+          <div class="art-item">
+            <div class="test"></div>
+            <h2 class="data">Website Portfolio</h2>
+            <p class="selections">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="buttons">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <p class="invite">See Project</p>
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="the-resume">
+        <div class="myresume">
+          <h3 class="h3-title">About me</h3>
+          <p class="about-paragraph">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button class="resume" type="button">Get My Resume</button>
+        </div>
+      </div>
+      <div>
+        <hr class="line222" />
+      </div>
+      <div class="work-on">
+        <div class="all-about">
+          <h3 class="best">
+            <img src="icons/Rectangle55.png" alt="rectangl" /> Language
+          </h3>
+          <ul class="list">
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+        <div class="all-about">
+          <h3 class="best">
+            <img src="icons/Rectangle56.png" alt="rectangle" /> Frameworks
+          </h3>
+          <ul class="list">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>Rspec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="all-about">
+          <h3 class="best">
+            <img src="icons/Rectangle57.png" alt="circle" /> Skills
+          </h3>
+          <ul class="list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section class="contact-form">
+      <div class="the-contact">
+        <h2 class="contact-p">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
```

Comment:
- [ ] As requested by the previous reviewer, please change this to a paragraph using a p tag
- [ ] As requested by the previous reviewer, please ensure that there is some margin between the paragraph and the name input field as it is in the template provided.  These fixes will make your design look more like the template provided.

Reply:
I gave the margin already but i don't know why you didn't see it